packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -206,16 +243,14 @@ const ChartMarkers = connect(({ modules, ui, client }) => ({
 }))(Markers);
 
 class ChartTradeClass extends React.Component {
-    state = {
-        active_category: null,
-    };
     bottomWidgets = ({ digits, tick }) => <ChartBottomWidgets digits={digits} tick={tick} />;
     topWidgets = ({ ...props }) => {
-        const { is_digits_widget_active, try_synthetic_indices } = this.props;
+        const { is_digits_widget_active, try_synthetic_indices, try_open_markets, open_market } = this.props;
+
```

Comment:
I think we can mix `open_market`, `try_synthetic_indices` and `try_open_markets` that makes it cleaner and more readable. so whenever `open_market` has value, it means we need to open the chart market selector. This way, the next developer needs to only follow the `open_market` property to find the process.

Reply:
I'm sure we can make some improvements but let's push this as it's been a long time we're working on it.